### PR TITLE
Policy: generate package-policy-2019-06

### PR DESCRIPTION
**Pull request by xccc-msft**

### CODE REVIEW

File: src/ResourceManagement/ResourceManager/Domain/PolicyDefinition/Definition/IDefinition.cs [hunk 1]
```diff
@@ -0,0 +1,111 @@
...
+    /// The stage of the PolicyDefinition definition allowing to specify policy rule.
+    /// </summary>
+    public interface IWithPolicyRule
+    {
+        /// <summary>
+        /// Specifies the policy rule of PolicyDefinition.
+        /// </summary>
+        /// <param name="policyRule">The policy rule of the PolicyDefinition.</param>
+        /// <return>The next stage of the definition.</return>
+        IWithCreate WithPolicyRule(Object policyRule);
```
Comment (weidongxu-microsoft) [inline src/ResourceManagement/ResourceManager/Domain/PolicyDefinition/Definition/IDefinition.cs]: Any idea why we would need to support a Object as input?

Comment (xccc-msft) [reply src/ResourceManagement/ResourceManager/Domain/PolicyDefinition/Definition/IDefinition.cs]: This is how it gets defined in [definition swagger](https://github.com/Azure/azure-rest-api-specs/blob/master/specification/resources/resource-manager/Microsoft.Authorization/stable/2019-06-01/policyDefinitions.json#L438)

Comment (weidongxu-microsoft) [reply src/ResourceManagement/ResourceManager/Domain/PolicyDefinition/Definition/IDefinition.cs]: Do you know what object this is? If you do not know, do you think SDK user would know?

Comment (xccc-msft) [reply src/ResourceManagement/ResourceManager/Domain/PolicyDefinition/Definition/IDefinition.cs]: Good catch. We could ask this question to [service side](https://docs.microsoft.com/en-us/rest/api/resources/policydefinitions/createorupdate)

Comment (yaohaizh) [reply src/ResourceManagement/ResourceManager/Domain/PolicyDefinition/Definition/IDefinition.cs]: What the value from the Java side?

Comment (xccc-msft) [reply src/ResourceManagement/ResourceManager/Domain/PolicyDefinition/Definition/IDefinition.cs]: In portal, the policy rule is json format. So it makes sense to have object parameters here. We have implemented object and json string both java and .net sides.

Comment (yaohaizh) [reply src/ResourceManagement/ResourceManager/Domain/PolicyDefinition/Definition/IDefinition.cs]: Please document this find on the API docs.

**Pull request merged**